packages/one-app-dev-bundler/__tests__/esbuild/plugins/one-app-index-loader.spec.js

```diff
@@ -221,16 +221,21 @@ describe('Esbuild plugin oneAppIndexLoader', () => {
       )).rejects.toThrow('one-app-bundler: Module must use `export default VariableName` syntax in index');
     });
 
-    it('should throw an exception if an injector removes the default export', async () => {
-      expect.assertions(1);
```

Comment:
removed assertions check?

Reply:
`expect.assertions(1);` should need to be used in `async` tests. Its purpose is to ensure legacy scenarios where `expect` was often called in catch blocks or `.then` promise chains where it's possible for the test to execute and not call the expect function. If an async test does not resolve it will fail.